docs/docsource/Operation-Architecture/Orbit_UI/Location_Management/Adding_a_Storage_Location.rst

```diff
@@ -11,31 +11,36 @@ Storage Locations dialog:
 
    |image1|  
 
-#.
-
    a. Enter a location name in the **Location Name** field using
       lowercase letters, numbers, and dashes.
 
       .. note::
 
-       Capital letters, spaces, and punctuation and diacritical
-       marks will result in an error message.
+         Capital letters, spaces, and punctuation or diacritical
+         marks will result in an error message.
 
    b. Select a location type from the **Location Type** pull-down menu.
-      You can choose Amazon S3, DigitalOcean Spaces, Google Cloud
-      Storage, Microsoft Azure Blob Storage, NFS Mount, Scality RING
-      with S3 Connector, Scality RING with sproxyd Connector, Wasabi, 
-      Ceph RADOS Gateway, or a Zenko Local Filesystem.
+      You can choose:
+
+      * Amazon S3
+      * DigitalOcean Spaces
+      * Wasabi
+      * Google Cloud Storage
+      * Microsoft Azure Blob Storage
+      * NFS Mount
+      * Scality RING with S3 Connector
+      * Scality RING with sproxyd Connector
+      * Ceph RADOS Gateway
+      * A Zenko local filesystem.
 
 #. Each storage location type has its own requirements. No security is
-   required for a local file system, but all clouds require
+   required for a local file system, but all public clouds require
    authentication information.
 
   .. note::
 
-    For release GA 1.0, Zenko has been capped at 1024 objects. Upward revision of
-    this cap is planned for subsequent product revisions.
-
+    Google Cloud Platform imposes a 1024-object cap. For all other 
+    backends, Zenko caps the number of objects at 10,000.
```

Comment:
Is this for open source version or something?

Reply:
The big problem here is that this should say "part" not "object".

As I understand it, these are architectural limits, though as Alex clarifies, the 1024-part cap comes from us, not GCP. The S3 protocol caps us at 10,000 parts.